packages/gatsby/content/advanced/questions-and-answers.md

```diff
@@ -38,44 +38,50 @@ Most projects will only face those three problems, which can all be fixed in a g
 
 ## Which files should be gitignored?
 
-If you're using Zero-Installs:
+Most commonly, the following patterns should be ignored:
```

Comment:
You'll find me annoying (sorry 😅), but I don't think there was an identified problem in need of a solving in this section - the main one being the missing link in the Getting Started, but that's just one link to add. I'd discard the changes in this files (unless you noticed a file entry being factually missing, in which case we can just add this one entry?) since frankly both old and new layouts have equivalent pros and cons.

Reply:
It's all fine. No worries. 🙂 

I made the changes based on my own experience with what I thought was missing. It feels like `Zero-Installs` are made prominent, but not even the `berry` repo uses it, and it's very hard to use in practice. I was misled into trying Zero-Installs by it being up top there I guess.

Just add a thumbs up on this comment and I'll revert it 🙂

Reply:
> not even the `berry` repo uses it

We do - our cache is checked into the repo, and various CI workflows don't run installs as a result

Reply:
But `.yarn/unplugged` and `build-state.yml` are not committed in the `berry` repo. 

I thought `Zero-Install` specifically means not having to run `yarn install` at all. 🤔